zmon-controller-app/src/main/resources/db/migration/V7__delete_update_entity_with_admin_authority.sql

```diff
@@ -0,0 +1,57 @@
+-- update to fix return of null if there are no checks at all.
+
+DROP FUNCTION IF EXISTS zzm_data.create_or_update_entity(entity_data text, teams text[], user_name text);
+DROP FUNCTION IF EXISTS zzm_data.delete_entity(id text, teams text[], user_name text);
+
+SET search_path TO zzm_api, public;
```

Comment:
By setting the search path like this, new functions, types, tables, ... are injected into zzm_api (just an FYI for above comment to why there is no prefix- schema in the create or replace function)